src/Utils/Strings.php

```diff
@@ -99,10 +99,13 @@ public static function substring(string $s, int $start, int $length = null): str
 
 
 	/**
-	 * Removes special controls characters and normalizes line endings and spaces in UTF-8 string.
+	 * Removes special controls characters and normalizes line endings and converts to NFC form and spaces in UTF-8 string.
```

Comment:
The sentence now makes no sense. How about sth like

Removes special controls characters and normalizes line endings, spaces and normal form to NFC in UTF-8 string

Reply:
👍 has been writing it in a car

Reply:
fixed in ec16e40